rotkehlchen/db/schema.py

```diff
@@ -786,6 +786,18 @@
 """  # noqa: E501
 
 
+DB_CREATE_CALENDAR_REMINDER = """
+CREATE TABLE IF NOT EXISTS reminder (
+    identifier INTEGER PRIMARY KEY NOT NULL,
+    event_id INTEGER NOT NULL,
+    amount INTEGER NOT NULL,
+    unit TEXT NOT NULL,
+    at_time TEXT,
+    FOREIGN KEY(event_id) REFERENCES calendar(identifier) ON DELETE CASCADE
+);
+"""
```

Comment:
plural ... just like we do with history events, trades etc.


```suggestion
DB_CREATE_CALENDAR_REMINDERS = """
CREATE TABLE IF NOT EXISTS calendar_reminders (
    identifier INTEGER PRIMARY KEY NOT NULL,
    event_id INTEGER NOT NULL,
    secs_before INTEGER NOT NULL,
    executed INTEGER NOT NULL CHECK (status IN (0, 1)),
    FOREIGN KEY(event_id) REFERENCES calendar(identifier) ON DELETE CASCADE
);
"""
```